lib/elixir_analyzer/test_suite/name_badge.ex

```diff
@@ -0,0 +1,22 @@
+defmodule ElixirAnalyzer.TestSuite.NameBadge do
+  @moduledoc """
+  This is an exercise analyzer extension module for the concept exercise NameBadge
+  """
+
+  use ElixirAnalyzer.ExerciseTest
+  alias ElixirAnalyzer.Constants
+
+  feature "Use an if macro" do
+    find :one
```

Comment:
AFAIK `:one` is not a valid `find` value
```suggestion
    find :any
```

Reply:
It is a valid value according to the [docs](https://github.com/exercism/elixir-analyzer/blob/main/docs/step-03/step-03.md) (line 208, GitHub is weird for linking markdown files I guess) but is means "match only one case" so `:any` is the best choice. Thank you.

Reply:
Oops, my bad :D I haven't seen it before.